### ISSUE TITLE
class_balanced_cross_entropy_loss

### ISSUE THREAD

**Issue by InstantWindy** [kmaninis/OSVOS-PyTorch#21]
Hello!
![image](https://user-images.githubusercontent.com/23608180/48401134-43cbac00-e763-11e8-87fc-8f869ad8164b.png)

I don't understand this formula. This formula is different from the sigmoid cross entropy loss function.Why do you calculate this way?Thank you !

**Comment by kmaninis** [comment]
Hi, please take a look here: https://www.tensorflow.org/api_docs/python/tf/nn/sigmoid_cross_entropy_with_logits

In short, it is a numerically stable version of the one that you are referring to.
